tests/kernel/mem_protect/userspace/src/main.c

```diff
@@ -324,7 +324,11 @@ static void pass_noperms_object(void)
 
 __kernel struct k_thread kthread_thread;
 
+#ifdef CONFIG_SOC_SERIES_MPS2
+#define STACKSIZE 1024
```

Comment:
just make this 1024 for everyone IMO, 512 was selected at random IIRC and it's not like MPU-enabled devices have really tiny amounts of RAM

Reply:
i agree.  i always wondered about that.  If people want to minimize that stuff they can add pare it back down sub <1k.